.expeditor/license_scout.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Does `chef-server` have these components? Do we need to update this or remove it?